README.adoc

```diff
@@ -1,4 +1,4 @@
-JBoss Enterprise Application Platform continuous delivery container image
+#JBoss Enterprise Application Platform continuous delivery container image
```

Comment:
This is wrong, although I suppose it is in the eap-cd-dev branch. I know I fixed that somewhere, maybe in the openshift image?

Reply:
https://github.com/jboss-container-images/jboss-eap-7-image/commit/672ef1c38b8912e43e75ffcf9adaf11ee5c5ecc5#diff-7bd4a925c695c2eb0eced3872b9b965f is what I was thinking of but that's the same as what you have here. I suspect it's wrong there, but anyway that all seems OT to this PR, which shouldn't be concerned with correcting issues in eap-cd-dev